hack/convert-image/convert_images.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
maybe we can have a better word than "convert image" ?

Reply:
Replace "convet image" with "image transfer", which means that copy the image from one registry to another.

Or maybe we can name it "image syncer"?